recipes/xarray-selafin/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "xarray_selafin" %}
+{% set version = "0.1.8" %}
+{% set sha256 = "7e1e952cd4f3c4dcfc5063508e528bec0d7bea0c208e005e3952d29a23e264a2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
```

Don't template single use variables. That obfuscates the recipe and doesn't really help. users.

```suggestion
  sha256: 7e1e952cd4f3c4dcfc5063508e528bec0d7bea0c208e005e3952d29a23e264a2
```